MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.CheckBox.xaml

```diff
@@ -82,7 +82,8 @@
                             <ColumnDefinition Width="Auto"/>
                             <ColumnDefinition Width="*"/>
                         </Grid.ColumnDefinitions>
-                        <Viewbox Width="18" Height="18"  VerticalAlignment="{TemplateBinding VerticalContentAlignment}">
+                        <Viewbox Width="18" Height="18"  VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
+                                 FlowDirection="LeftToRight">
```

Comment:
The same thing as Icons happen here. the check mark inside the Checkbox should look the same on RTL or LTR.